coral/src/app/features/components/filters/TeamFilter.tsx

```diff
@@ -3,14 +3,17 @@ import { useQuery } from "@tanstack/react-query";
 import { useFiltersContext } from "src/app/features/components/filters/useFiltersContext";
 import { getTeams } from "src/domain/team/team-api";
 
-function TeamFilter() {
-  const { data: topicTeams } = useQuery(["get-teams"], {
+type TeamFilterProps = {
+  useTeamName?: boolean;
```

Comment:
Did you look into making some kind of map betweem `teamName` and `teamId` somewhere so we don't have to add this parameter to this filter? :o

I understand that it's a little tricky that the endpoint wants `teamName` as a param...

Reply:
We could change it in the backend, but otherwise `TeamFilter `needs to return the `teamName` as it's the only component that could own the map. At least I landed there, not sure if I overlooked something.

Reply:
I think all the other options are probably more complexity than this.

The best option would be to change the endpoint as you said, maybe that could be an issue? "Use entity IDs always instead entity names".

Reply:
Good idea, I'll create a follow up issue 🏃